File: core/src/main/java/com/sys1yagi/swipe/core/tool/PabloPicasso.java
package com.sys1yagi.swipe.core.tool;

import com.squareup.picasso.Picasso;

import android.content.Context;

public class PabloPicasso {

    public Picasso picasso;

    public PabloPicasso(Context context) {
        picasso = Picasso.with(context);
    }


}

File: core/src/main/java/com/sys1yagi/swipe/core/tool/PabloPicasso.kt
package com.sys1yagi.swipe.core.tool

import com.squareup.picasso.Picasso

import android.content.Context

class PabloPicasso(context: Context) {

    var picasso: Picasso

    init {
        picasso = Picasso.with(context)
    }


}
